2020-01-Winter/1_linked_lists/.gitignore

```diff
@@ -0,0 +1,3 @@
+CMakeLists.txt
+
+cmake-build-debug
```

Comment:
ICYMI, there's a `.gitignore` at the repository root [here](https://github.com/UWB-ACM/CTCI/blob/master/.gitignore)